tools/db_bench_tool.cc

```diff
@@ -4663,6 +4704,15 @@ class Benchmark {
       return std::numeric_limits<uint64_t>::max();
     }
 
+    uint64_t Fetch(uint64_t index) {
+      if (mode_ == UNIQUE_RANDOM) {
+        assert(index < values_.size());
+        return values_[index];
+      }
+      assert(false);
```

Comment:
I think it would be simpler / clearer to use `assert(mode_ == UNIQUE_RANDOM);` above and skip this section. (This isn't production code where we should reasonably try to avoid undefined behavior in case of broken invariants.)